src/effects/visibleeffectslist.cpp

```diff
@@ -68,16 +72,19 @@ void VisibleEffectsList::readEffectsXml(
         }
     }
 
-    if (!list.isEmpty()) {
-        setList(list);
-    } else {
+    if (list.isEmpty() && readVisible) {
+        // Show all built-in effects by default.
         setList(pBackendManager->getManifestsForBackend(EffectBackendType::BuiltIn));
```

Comment:
Is this condition still relevant? We will add  new effects anyway in the outside code.